src/guide/refund_payment.md

```diff
@@ -1,10 +1,10 @@
 # Refunding Payments
 
-The SDK handles refunding of payments that have failed automatically, though sometimes it is necessary for the user to intervene in order to recover their funds.
+The SDK in most cases handles refunding of payments that have failed automatically. But when receiving Bitcoin payments, the refund of a failed swap has to be managed manually.
```

Comment:
NIT: perhaps be more concrete?
"The SDK handles refunding of failed payments automatically except when receiving Bitcoin payments where the refund of a failed swap has to be managed manually"

Reply:
Agreed, done